### DISCUSSION TITLE
Merge #208 didn't pass circle CI

### DISCUSSION THREAD

**Discussion by mh4647** [tony19/logback-android#234] [Q&A] [answered]
Hi,
The merge https://github.com/tony19/logback-android/commit/e65d6405bc1024b4d8d21089e95f2ca2f7052f7d
has an issue with Circle CI android 29 installation.

**Comment by tony19** [comment, marked as answer]
Thanks for the heads up! I cannot reproduce the issue. I think that can be ignored.
